dbms/src/Interpreters/Aggregator.cpp

```diff
@@ -661,7 +662,10 @@ AggregatedDataVariants::Type Aggregator::chooseAggregationMethodInner()
     if (params.keys_size == 1 && types_not_null[0]->isFixedString())
         return AggregatedDataVariants::Type::key_fixed_string;
 
-    return ChooseAggregationMethodFastPath(params.keys_size, types_not_null, params.collators);
```

Comment:
Do we still need to keep the implementation of ChooseAggregationMethodFastPath?

Reply:
Actually it can be removed, i will remove it.

Reply:
done